src/main/java/uk/gov/digital/ho/hocs/client/casework/CaseworkClient.java

```diff
@@ -7,10 +7,7 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
 import uk.gov.digital.ho.hocs.application.RestClient;
-import uk.gov.digital.ho.hocs.client.casework.dto.ComplaintCorrespondent;
-import uk.gov.digital.ho.hocs.client.casework.dto.UpdateCaseworkCaseDataRequest;
-import uk.gov.digital.ho.hocs.client.casework.dto.UpdateStageTeamRequest;
-import uk.gov.digital.ho.hocs.client.casework.dto.UpdateStageUserRequest;
+import uk.gov.digital.ho.hocs.client.casework.dto.*;
```

Comment:
There will be a setting to stop this collapsing happening from within your IDE as we shouldn't be importing entire packages into classes 🙂

Reply:
I've changed the settings in IDE and fixed imports.